src/test/e2e/00_use_cli_test.go

```diff
@@ -216,21 +217,31 @@ func TestUseCLI(t *testing.T) {
 	t.Run("zarf tools yq should function appropriately across different uses", func(t *testing.T) {
 		t.Parallel()
 
-		file := "src/test/packages/00-yq-checks/file1.yaml"
-		otherFile := "src/test/packages/00-yq-checks/file2.yaml"
+		tmpdir := t.TempDir()
+		originalPath := "src/test/packages/00-yq-checks"
+
+		originalFile := filepath.Join(originalPath, "file1.yaml")
+		originalOtherFile := filepath.Join(originalPath, "file2.yaml")
+
+		file := filepath.Join(tmpdir, "file1.yaml")
+		otherFile := filepath.Join(tmpdir, "file2.yaml")
+
+		copy.Copy(originalFile, file)
+		copy.Copy(originalOtherFile, otherFile)
```

Comment:
We should handle the error returned from `copy.Copy()`

<img width="863" alt="error" src="https://github.com/defenseunicorns/zarf/assets/87675701/68ee6625-96bb-4e4b-a8ec-2c39a6a94b94">

Reply:
Also need to run `go mod tidy`